### PR TITLE
add missing namespace traits

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- move trait `GetPusher` to namespace traits
- move trait `GetIonizer` to namespace traits

### CODE REVIEW

**Comment by Heikman** [comment]
shouldn't all the traits in picongpu/particles/traits reside in namespace picongpu::particles::traits like it is in libPMacc?

File: src/picongpu/include/particles/traits/GetIonizer.hpp [hunk 1]
```diff
@@ -34,6 +34,8 @@
 namespace picongpu
 {
 
+namespace traits
```
Comment (ax3l) [inline src/picongpu/include/particles/traits/GetIonizer.hpp]: yes, should actually be in `picongpu::particles::traits`

**Comment by ax3l** [comment]
@psychocoderHPC will you update that in a follow-up?

File: src/picongpu/include/particles/traits/GetIonizer.hpp (hunk 1 above)
Comment (psychocoderHPC) [inline src/picongpu/include/particles/traits/GetIonizer.hpp]: all other traits in this folder are in `picongpu::traits` therefore I only change the different to keep the over all change small

Comment (psychocoderHPC) [inline src/picongpu/include/particles/traits/GetIonizer.hpp]: open issue #1246 for this change

Comment (ax3l) [inline src/picongpu/include/particles/traits/GetIonizer.hpp]: k, so a follow-up could change them all at once.

**Comment by psychocoderHPC** [comment]
I opened a issue #1246 for it

**Pull request merged**